.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: python
+language: generic
```

Comment:
I think this is making the 'lint' job fail

Reply:
Thanks @crusaderky. Fixed